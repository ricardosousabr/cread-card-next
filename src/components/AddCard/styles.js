import styled from 'styled-components'

export const Container = styled.div`
  display: flex;
  align-items: center;
  justify-content: center;
  flex-direction: column;
  width: 100%;
  height: 100vh;
`

export const BoxTitle = styled.div`
  display: flex;
  align-items: center;
  justify-content: center;
`

export const Title = styled.h1`
  font-size: 20px;
  color: #ffffff;
`
