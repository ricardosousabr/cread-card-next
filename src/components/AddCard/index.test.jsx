/**
 * @jest-environment jsdom
 */

import React from 'react'
import AddCard from './index'
import renderer from 'react-test-renderer'

test('HomePage snapshot', () => {
  const component = renderer.create(<AddCard />)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
