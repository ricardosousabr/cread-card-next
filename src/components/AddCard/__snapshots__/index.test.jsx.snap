// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage snapshot 1`] = `
<div
  className="styles__Container-sc-1la1y2-0 itsoVm"
>
  <div
    className="styles__BoxTitle-sc-1la1y2-1 eBIxUJ"
  >
    <div>
       
      <h1
        className="styles__Title-sc-1la1y2-2 kdSUAQ"
      >
        Add your credit card
      </h1>
    </div>
    <div>
       
      <svg
        height="32"
        viewBox="0 0 24 24"
        width="32"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M22.0049 10V20C22.0049 20.5523 21.5572 21 21.0049 21H3.00488C2.4526 21 2.00488 20.5523 2.00488 20V10H22.0049ZM22.0049 8H2.00488V4C2.00488 3.44772 2.4526 3 3.00488 3H21.0049C21.5572 3 22.0049 3.44772 22.0049 4V8ZM15.0049 16V18H19.0049V16H15.0049Z"
          fill="#7d7676"
        />
      </svg>
    </div>
  </div>
  <div>
    <div>
      <div>
        <div
          className="styles__ContanerCard-sc-4f1d5k-0 hZqvfl"
        >
          <div>
            <p
              className="styles__NumberCard-sc-4f1d5k-1 kgiWKl"
            />
          </div>
          <div>
            <div
              className="styles__TitleDate-sc-4f1d5k-5 eaODtn"
            >
              Nome do titular
            </div>
            <p
              className="styles__CardholderName-sc-4f1d5k-2 dVFrPG"
            />
          </div>
          <div>
            <div
              className="styles__TitleDate-sc-4f1d5k-5 eaODtn"
            >
              Expiração
            </div>
            <p
              className="styles__CardExpiration-sc-4f1d5k-3 jUASOX"
            />
          </div>
          <div>
            <div
              className="styles__TitleDate-sc-4f1d5k-5 eaODtn"
            >
              CVC
            </div>
            <p
              className="styles__CardCvc-sc-4f1d5k-4 hTCMql"
            />
          </div>
        </div>
      </div>
      <div>
        <form
          action=""
          className="styles__ContainerForm-sc-wedmof-0 dPZUAw"
        >
          <div>
            <p
              className="styles__Text-sc-wedmof-4 iCLgfV"
            >
              Número do cartão
            </p>
            <input
              className="styles__InputLarge-sc-wedmof-1 cpyqzR"
              onChange={[Function]}
              type="number"
            />
          </div>
          <div>
            <p
              className="styles__Text-sc-wedmof-4 iCLgfV"
            >
              Nome do titular
            </p>
            <input
              className="styles__InputLarge-sc-wedmof-1 cpyqzR"
              onChange={[Function]}
              type="text"
            />
          </div>
          <div
            className="styles__BoxInputSmall-sc-wedmof-2 lnSgfT"
          >
            <div>
              <p
                className="styles__Text-sc-wedmof-4 iCLgfV"
              >
                Expiração
              </p>
              <input
                className="styles__InputSmall-sc-wedmof-3 SFUAC"
                onChange={[Function]}
                type="date"
              />
            </div>
            <div>
              <p
                className="styles__Text-sc-wedmof-4 iCLgfV"
              >
                CVC
              </p>
              <input
                className="styles__InputSmall-sc-wedmof-3 SFUAC"
                onChange={[Function]}
                type="number"
              />
            </div>
          </div>
          <div
            className="styles__BoxButton-sc-wedmof-5 jprGjP"
          >
            <button
              className="styles__ButtonAddCard-sc-11nj3cj-0 eZHHSY"
            >
              Adicionar
            </button>
          </div>
        </form>
      </div>
    </div>
  </div>
</div>
`;
